hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -2884,4 +2901,41 @@ public int getMaxCompactedStoreFileRefCount() {
       ? maxCompactedStoreFileRefCount.getAsInt() : 0;
   }
 
+  @Override
+  public long getReadRequestsFromStoreCount() {
+    return getRequestsFromStore.sum();
+  }
+
+  @Override
+  public long getGetRequestsCountFromMemstore() {
+    return getRequestsFromMemstore.sum();
+  }
+
+  @Override
+  public long getGetRequestsCountFromFile() {
+    return getRequestsFromFile.sum();
+  }
+
+  void incrGetRequestsFromStore() {
+    getRequestsFromStore.increment();
```

Comment:
The one direclty inder HStore is used by the Region level and Table level aggregators which deals with HStore. This gets printed periodically. The other one is at the MetricsStore level which is the real time one. For every request it will be displayed at JMX MBean level.